frost-coordinator/Cargo.toml

```
@@ -20,6 +20,10 @@ tracing-subscriber = { workspace = true }
 frost-signer = { version = "0.0.1", path = "../frost-signer" }
 serde = { version = "1.0", features = ["serde_derive"] }
 
+[dev-dependencies]
+relay-server = { version = "0.1.0", path = "../relay-server" }
```

I suggest omitting the version for local dependencies in the same repo.
```suggestion
relay-server = { path = "../relay-server" }
```

---

Good catch!

---

Done.